openpype/hosts/max/api/preview_animation.py

```diff
@@ -36,9 +36,12 @@ def viewport_camera(camera):
         original = rt.getNodeByName(camera)
     review_camera = rt.getNodeByName(camera)
     try:
+        if rt.viewport.getLayout() != rt.Name("layout_1"):
+            rt.viewport.setLayout(rt.Name("layout_1"))
         rt.viewport.setCamera(review_camera)
         yield
     finally:
+        rt.viewport.ResetAllViews()
```

Comment:
This context manager should actually be reverting to the layout afterwards instead of `resetting`?

```
original_layout = rt.viewport.getLayout()
```
etc.